packages/cli/package.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "@synthql/cli",
+    "version": "0.0.1",
+    "repository": {
+        "type": "git",
+        "url": "git@github.com:synthql/synthql.git"
+    },
+    "license": "MIT",
+    "scripts": {
+        "build": "yarn tsc",
+        "publish:minor": "yarn publish --access public --no-git-tag-version",
+        "format": "yarn prettier --config ../../prettier.config.js --write ./src/"
```

Comment:
This is missing a `test:ci` and `test` script

Reply:
Adding this now using the Vite commands from the `queries` package.